### PR TITLE
fixes for trav dog

### PR DESCRIPTION
**Pull request by timatron**

### CODE REVIEW

File: php/class-fieldmanager-zone-field.php [hunk 1]
```diff
@@ -155,7 +155,7 @@ public function get_current_posts_json( $ids ) {
 						'post_type' => 'any',
 						'orderby' => 'post__in',
 						'order' => 'asc',
-						'posts_per_page' => 100, // arbitrarily high limit
+						'posts_per_page' => 50, // highest limit to not flag sniffer
```
Comment (romancandlethoughts) [inline php/class-fieldmanager-zone-field.php]: hold off on this.

**Comment by romancandlethoughts** [comment]
hold off on the post limit for now, I will push a fix up patch shortly. otherwise 🎱

**Comment by romancandlethoughts** [comment]
pinging @mboynes to pick up review

File: php/class-fieldmanager-zone-field.php [hunk 2]
```diff
@@ -155,7 +156,8 @@ public function get_current_posts_json( $ids ) {
 						'post_type' => 'any',
 						'orderby' => 'post__in',
 						'order' => 'asc',
-						'posts_per_page' => 100, // arbitrarily high limit
+						'posts_per_page' => count( $ids ),
```
Comment (timatron) [inline php/class-fieldmanager-zone-field.php]: still wont work if its over 50

Comment (timatron) [inline php/class-fieldmanager-zone-field.php]: worth doing a simple batch loop, then passing that to format_posts?

Comment (romancandlethoughts) [inline php/class-fieldmanager-zone-field.php]: why wouldn't this work over 50? @timatron

**Comment by mboynes** [comment]
:+1:

**Comment by romancandlethoughts** [comment]
:party:

**Pull request merged**